### DISCUSSION TITLE
Why can't my ESP32-C3 use this library?

### DISCUSSION THREAD

**Discussion by SFNFIH** [HomeSpan/HomeSpan#560] [General]
The chip module on the development board I bought is ESP32-C3-MINI-1. After burning the firmware, there is no response, even the serial monitor does not give any feedback. What is the problem? Burn the same code to the ESP32-WROOM-32E module and it will run normally.

**Comment by HomeSpan** [comment]
Does it fail if you run Example 1 without any modifications?
